menu.js

```diff
@@ -52,5 +53,350 @@ link3.addEventListener('click', () => {
+
+  seeDiv.classList.add('see-div');
+  btnb1.classList.add('See-project', 'seeAlt');
+  btnb2.classList.add('See-project', 'seeAlt');
+  btnb1.innerText = 'See Live';
+  btnb2.innerText = 'See Source';
+
+  imagelive.alt = 'image';
+  imageSrc.alt = 'image';
+
+  imagelive.src = './ICONS/see-live.png';
+  imageSrc.src = './ICONS/see-source.png';
+
+  // Attaching current variable to DOM tree
+
+  body.append(popupSection);
+  popupSection.append(popupHead, wrapper1, popupimagVar1, popupimgaVar2, WrapperDiv);
+  popupHead.append(tonicTitleMob, tonicTitleDes, popCloser);
+  wrapper1.append(poplist1, poplist2, poplist6, poplist3, poplist4, poplist7, poplist5);
+  poplist1.append(linka1);
+  poplist2.append(linka2);
+  poplist6.append(separator1);
+  poplist3.append(linka3);
+  poplist4.append(linka4);
+  poplist7.append(separator2);
+  poplist5.append(linka5);
+
+  WrapperDiv.append(footingpar, buttonswrap);
+  buttonswrap.append(wrapper2, seeDiv);
+
+  wrapper2.append(popli1, popli2, popli3);
+  popli1.append(btna1);
+  popli2.append(btna2);
+  popli3.append(btna3);
+
+  seeDiv.append(btnb1, btnb2);
+
+  // popup-window manipulation
+  // const popupWindow = document.getElementById('popupWindoW');
+  // const BtnclosePopup = document.getElementById('close-popup');
+
+  SeeProjectButton.addEventListener('click', () => {
+    popupSection.classList.remove('popup-hide');
+    section1.classList.add('blur');
+    mainContainer.classList.add('blur');
+    mainContainer.classList.add('backblur');
+    headernav.classList.add('hide');
+  });
+
+  popCloser.addEventListener('click', () => {
+    popupSection.classList.add('popup-hide');
+    section1.classList.remove('blur');
+    mainContainer.classList.remove('blur');
+    mainContainer.classList.remove('backblur');
+    headernav.classList.remove('hide');
+  });
+}
+
+// creation of projects data
+
+const ProjectWrapper = [{
+  title: 'Tonic', content: 'A daily selection of privately personalized reads; no accounts or sign-ups required.', imageDesktop: './IMG/Snap1.png', imageMobile: './IMG/Snapshoot-Portfolio-1.svg', order1: '1', order2: '2',
+},
+{
+  title: 'Multi-Post Stories', content: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.', imageDesktop: './IMG/Snap2.png', imageMobile: './IMG/Snapshoot-Portfolio2.svg', order1: '2', order2: '1',
+},
+{
+  title: 'Facebook _360', content: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.', imageDesktop: './IMG/Snap3.png', imageMobile: './IMG/Snapshoot-Portfolio-1.svg', order1: '1', order2: '2',
+},
+{
+  title: 'ber Navigation', content: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.', imageDesktop: './IMG/Snap4.png', imageMobile: './IMG/Snapshoot-Portfolio2.svg', order1: '2', order2: '1',
+}];
+
```

Comment:
- [ ] [optional] Good job on the design but for this project you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible.
- Please change the design for the listed technologies for the project, they are not supposed to be buttons, and make sure it matches the design on the template.
- Please place the icons inside the buttons of `"see live "` and `"see source"`, and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/162277617-f6d581ff-3816-4a05-9391-ecd65127c5bd.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/162277648-6dfeac24-2410-4f0d-8fac-175644d5af1a.png)